tests/migration/version_recover_origin_system.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
```

Comment:
Thinking about the name of version_recover_origin_system.pm , for me 'recover' is not suitable because no 'recover' thing here, I suggest to use 'set' instead of 'recover'. WDYT?

Reply:
For the name, 'switch' is very suitable but it already used in old module, maybe 'reset' or 'set' is somehow like the meaning. Anyway, we'd better find a suitable name here.

Reply:
Changed to `migration/version_reset_origin_system`